WalletWasabi.Gui/Converters/WindowStateAfterSartJsonConverter.cs

```diff
@@ -20,6 +20,13 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
 			try
 			{
 				// If minimized, then go with Maximized, because at start it shouldn't run with minimized.
+				if (reader.Value is string s)
```

Comment:
IMO this could be simplified as follow:
```c#
var value = reader.Value as string;
if(string.IsNullOrWhiteSpace)
{
     return WindowState.Maximized;
}
```
In this way the next if statement is also redundant.

```